### DISCUSSION TITLE
Where should dkim(.pub).key go?

### DISCUSSION THREAD

**Discussion by mrghxst** [simple-login/app#692] [Self-hosting question] [answered]
Where should dkim(.pub).key go after I generated them ??

I just let them sit in home/ubuntu and then on this step:
Before running the webapp, you need to prepare the database by running the migration:

I get the error:
   with __ctx.ensure_object(ScriptInfo).load_app().app_context():
  File "/usr/local/lib/python3.7/site-packages/flask/cli.py", line 392, in load_app
    app = locate_app(self, import_name, None, raise_if_not_found=False)
  File "/usr/local/lib/python3.7/site-packages/flask/cli.py", line 240, in locate_app
    __import__(module_name)
  File "/code/wsgi.py", line 1, in <module>
    from server import create_app
  File "/code/server.py", line 28, in <module>
    from app import paddle_utils, s3, config
  File "/code/app/paddle_utils.py", line 18, in <module>
    from app.config import PADDLE_PUBLIC_KEY_PATH, PADDLE_VENDOR_ID, PADDLE_AUTH_CODE
  File "/code/app/config.py", line 156, in <module>
    with open(DKIM_PRIVATE_KEY_PATH) as f:
**IsADirectoryError: [Errno 21] Is a directory: '/dkim.key'**

I was thinking this is because they are not in the correct directory but I am not really sure about this !!
Thanks :heart:

**Comment by nguyenkims** [comment, marked as answer]
@mrghxst can you do a `ls` on the folder to see if the `dkim.key` and `dkim.pub.key` are there? They should be files and not directories.
